IotControlCenterUi/src/Components/LightingPageComponents/LightingComponents/SceneButtonBox.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, Grid } from "@mui/material";
+import React, { FC } from "react";
+import config from "../../../config";
+import {
+  gridContainerStyle,
+  gridItemStyle,
+} from "../../../Styles/DialogStyles";
+import { LightingRequest } from "../../../types";
+import { post } from "../../../utils";
+import IOperationButton from "../../OperationButton";
+
+const buttonBoxStyle = {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+  height: "100%",
+  backgroundColor: "#2C2C2E",
+  width: "90%",
+  borderRadius: "10px",
+  paddingBottom: "10px",
+};
+
+const onClickOcean = () => {
+  const sceneRequest: LightingRequest = {
+    operation: "ocean",
+  };
+  post(config.OCEAN_SCENE_ENDPOINT, sceneRequest);
```

Comment:
There shouldn't be an endpoint for each scene. It should be a single endpoint with each route concatenated onto it. e.g.:
`config.SCENE_ENDPOINT + "/ocean"` or `config.SCENE_ENDPOINT + "/rose"`

Reply:
Same for all other onclick functions below

Reply:
For this, you'll also have to change the `config.ts` routes so that there's only one route for all scene endpoints

Reply:
It’s funny because there is only one route in config.ts. I must've forgot to update these.

Reply:
Imagine grinding a PR at cafe campus